dask/array/numpy_compat.py

```diff
@@ -8,6 +8,7 @@
 _numpy_117 = LooseVersion(np.__version__) >= "1.17.0"
 _numpy_118 = LooseVersion(np.__version__) >= "1.18.0"
 _numpy_120 = LooseVersion(np.__version__) >= "1.20.0"
+_numpy_121 = LooseVersion(np.__version__) >= "1.21.0"
```

Comment:
This is only used in a comment so far, but it really helps to have it for greppability